doc/source/poppy_config.rst

```diff
@@ -0,0 +1,72 @@
+
+=========================
+Poppy Configuration Notes
+=========================
+
+Description
+-----------
+
+Please find below various poppy conf file configuration beside which in comments you will find the necessity of those configurations.
+These are some of the important configurations and also some regular configurations, not all the configuration information are explained below.
+
+Default configuration
+---------------------
+
+Make sure to keep the logging.conf in the correct directory
+
+[DEFAULT]
```

Comment:
Any `.ini` code should be formatted in a code block.

With rst, it can be done like:

```
.. code:: ini

  # (notice the indent)
  [default]
  log_path=/var/log/log.log
  other_settting=false
```